.travis.yml

```diff
@@ -32,13 +32,13 @@ rvm:
 # policy](http://guides.rubyonrails.org/maintenance_policy.html)
 gemfile:
   - gemfiles/rails_5.2.rb
-  - gemfiles/rails_6.0.rb
```

Comment:
I believe this shouldn't be removed, to ensure that we still have compatibility with Rails 6.0.x.

Reply:
Goo point! I wasn't sure if the priority was to be completist or keep it to 2 Gemfiles (to conserve travis credits, etc.)

Reply:
> .. to be completist or keep it to 2 Gemfiles (to conserve travis credits, etc.)

Let's prioritize coverage (being completist) for now. If the CI build becomes too slow in the future, we can revisit the decision.